polygon-id/id3/id3.jsonld

```diff
@@ -1,34 +1,337 @@
+                  "ProfileName": {
+                    "@id": "polygon-vocab:ProfileName",
+                    "@type": "xsd:string"
+                  },
+                  "ProfileVersion": {
+                    "@id": "polygon-vocab:ProfileVersion",
+                    "@type": "xsd:integer"
+                  },
+                  "ProfileRevision": {
+                    "@id": "polygon-vocab:ProfileRevision",
+                    "@type": "xsd:integer"
+                  },
+                  "ProfileState": {
+                    "@id": "polygon-vocab:ProfileState",
+                    "@type": "xsd:string"
+                  },
+                  "Score": {
+                    "@id": "polygon-vocab:Score",
+                    "@type": "xsd:integer"
+                  },
+                  "BandText": {
+                    "@id": "polygon-vocab:BandText",
+                    "@type": "xsd:string"
+                  },
+                  "Country": {
+                    "@id": "polygon-vocab:Country",
+                    "@type": "xsd:string"
+                  },
+                  "ResultCodes": {
+                    "@context": {},
+                    "@id": "polygon-vocab:ResultCodes"
+                  }
+                },
+                "@id": "polygon-vocab:authenticateSPResult"
+              }
+            },
+            "@id": "polygon-vocab:originalData"
           },
-          "personalDetails":{
-            "@id":"polygon-vocab:personalDetails",
-            "@type":"@json"
+          "personalDetails": {
+            "@context": {
+              "citizenship": {
+                "@id": "polygon-vocab:citizenship",
+                "@type": "xsd:string"
+              },
+              "documentType": {
+                "@id": "polygon-vocab:documentType",
+                "@type": "xsd:string"
+              },
+              "firstName": {
+                "@id": "polygon-vocab:firstName",
+                "@type": "xsd:string"
+              },
+              "lastName": {
+                "@id": "polygon-vocab:lastName",
+                "@type": "xsd:string"
+              },
+              "gender": {
+                "@id": "polygon-vocab:gender",
+                "@type": "xsd:string"
+              },
+              "addressLine1": {
+                "@id": "polygon-vocab:addressLine1",
+                "@type": "xsd:string"
+              },
+              "addressLine2": {
+                "@id": "polygon-vocab:addressLine2",
+                "@type": "xsd:string"
+              },
+              "birthday": {
+                "@id": "polygon-vocab:birthday",
+                "@type": "xsd:dateTime"
+              },
+              "placeOfBirth": {
+                "@id": "polygon-vocab:placeOfBirth",
+                "@type": "xsd:string"
+              },
+              "countryOfResidence": {
+                "@id": "polygon-vocab:countryOfResidence",
+                "@type": "xsd:string"
```

Comment:
maybe this could be enum? well not sure, we have to try it.. it is fine if we leave it as string, we have enum in the schema

Reply:
i could try it -- i'm also seeing this on searching for `+jsonld @enum`

https://dotnetrdf.org/docs/3.1.x/api/VDS.RDF.JsonLd.Syntax.JsonLdContainer.html

by my reading, there is no such field as "@enum" in the JsonLdContainer.  Even though the set of such fields is an enum. :)

Reply:
also note that even with fields that have "enum" defined in the json schema, they also have a type defined.

and the word "enum" does not even appear in this document defining the jsonld schema:

https://www.w3.org/TR/json-ld11/